db/db_impl.h

```diff
@@ -561,6 +561,13 @@ class DBImpl : public DB {
 
   const SnapshotList& snapshots() const { return snapshots_; }
 
+  const std::vector<SequenceNumber> CopySnapshots(
+      SequenceNumber* oldest_write_conflict_snapshot = nullptr,
+      const SequenceNumber& max_seq = kMaxSequenceNumber) const {
+    InstrumentedMutexLock l(mutex());
+    return snapshots().GetAll(oldest_write_conflict_snapshot, max_seq);
```

Comment:
Since now GetAll() is called more frequently, consider to fill a vector passed in to avoid memory allocation. Doesn't have to be done in this PR though.